README.md

```diff
@@ -23,6 +23,7 @@ If it wont load, open an issue :)
 | -h | Displays the help menu in the app |
 | -w | Disables randomization of weapons. Enabled by default |
 | -p | Disables randomization of megaman pallete. Enabled by default |
+| -roll | Applies Roll-Chan graphics patch (Credit Zynk). Disabled by default |
```

Comment:
consider changing this to a +roll, since -whatever is about disabling something while this applies something

Reply:
Ah that's a good point, I'll keep that in mind for future additions as well!

I did plan on having Roll-Chan as a option for the randomizer, which is why I went ahead documentation. But sense the branch is focused on IPS support, I could remove Roll-Chan for now and add it in another PR?

Reply:
Changed instances of "-roll" to "+roll" in the meantime